pkg/cmd/cli/cmd/rsync/rsync.go

```diff
@@ -229,7 +237,110 @@ func (o *RsyncOptions) Validate() error {
 
 // RunRsync copies files from source to destination
 func (o *RsyncOptions) RunRsync() error {
```

Comment:
and i was hoping to avoid building up a list of what has changed and using some alternate invocation of rsync to process just that list.